### PR TITLE
Add descriptions to sync node and AI controller

### PR DESCRIPTION
**Pull request by metinc**
This adds descriptions to the props to make the plugin more accessible to new users.

Examples:

image

image

### CODE REVIEW

File: addons/godot_rl_agents/controller/ai_controller_2d.gd
Comment (metinc) [inline addons/godot_rl_agents/controller/ai_controller_2d.gd:9]: Not sure what RECORD_EXPERT_DEMOS does so I don't have a description here.

File: addons/godot_rl_agents/sync.gd
Comment (metinc) [inline addons/godot_rl_agents/sync.gd:2]: Somehow I had to add a class name to make the descriptions for ControlModes show up in the editor.

File: addons/godot_rl_agents/controller/ai_controller_2d.gd
Comment (Ivan-267) [reply addons/godot_rl_agents/controller/ai_controller_2d.gd:9]: It's an option for recording expert demonstrations (i.e. playing the game manually to show the agent what to do). The data includes the obs and actions taken. However, a bit more code needs to be implemented when using this option.

There is a wip course bonus unit for this: https://github.com/huggingface/deep-rl-class/pull/537

Comment (metinc) [reply addons/godot_rl_agents/controller/ai_controller_2d.gd]: Thanks, I added the description.

File: addons/godot_rl_agents/sync.gd
Comment (Ivan-267) [reply addons/godot_rl_agents/sync.gd:2]: I'm also not seeing the suggestions with Godot 4.3 beta 2 even with the class_name, I will test it more, maybe removing `.godot` folder will help after updating the plugin.

image

File: addons/godot_rl_agents/sync.gd
Comment (Ivan-267) [inline addons/godot_rl_agents/sync.gd:12]:
```suggestion
## Action will be repeated for n frames (Godot physics steps). 
```

File: addons/godot_rl_agents/sync.gd
Comment (Ivan-267) [inline addons/godot_rl_agents/sync.gd:13]:
```suggestion
```

File: addons/godot_rl_agents/sync.gd
Comment (Ivan-267) [inline addons/godot_rl_agents/sync.gd:14]:
```suggestion
```

File: addons/godot_rl_agents/controller/ai_controller_3d.gd
Comment (Ivan-267) [inline addons/godot_rl_agents/controller/ai_controller_3d.gd:12]:
```suggestion
## The path to a trained .onnx model file to use for inference (overrides the path set in sync node).
```

File: addons/godot_rl_agents/controller/ai_controller_2d.gd
Comment (Ivan-267) [inline addons/godot_rl_agents/controller/ai_controller_2d.gd:12]:
```suggestion
## The path to a trained .onnx model file to use for inference (overrides the path set in sync node).
```

**Review by Ivan-267 — APPROVED** [review]
Looks great overall, thanks for adding this. I've added a few small suggestions, after considering those feel free to merge the PR.

This may not work in some cases for the sync node in Godot without changing the class name as we discussed, but it will likely work in future versions.

**Comment by metinc** [comment]
Thanks! Feel free to merge it. I don't have write access on this repo.

**Pull request merged**
